README.md

```diff
@@ -1,5 +1,17 @@
 # trace-server-protocol
```

Comment:
Commit message(s) and PR header are missing a `Signed-off-by` footer.

Reply:
Addendum: commit message's body line needs a wrap to max 72 chars, per guideline.